### PR TITLE
update User guide

### PR DESCRIPTION
**Pull request by cpandhi**
Updates to AC guide:

- intro wording
- options
- filters
- highlights

One update to archives.json - change the name from Latest archive to Latest deployment

### CODE REVIEW

**Comment by cpandhi** [comment]
@drjoho the AC guide changes are ready to merge

**Comment by cpandhi** [comment]
@sharitrewin great if you can review the updates to the AC guide

**Comment by cpandhi** [comment]
@eirckr found 1 typo and have fixed it. Something else it making the test branch fail. @drjoho  can you please check.

**Comment by sharitrewin** [comment]
@cpandhi did you still want me to review this PR?  I took a quick look and I have a couple of suggestions but I'm not listed as a reviewer.  Do you want me to add them as comments here instead?

**Comment by cpandhi** [comment]
@sharitrewin yes, please. I have added you as a reviewer.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 1]
```diff
@@ -507,6 +504,12 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
                     available. All views show the same set of issues.
                   </p>
                 </li>
+                <li>
+                  <p style={{ marginTop: "0rem" }}>
+                    To <strong>filter</strong> issues by violations, needs review or by recommendations, press any issue type toggle button in the scan results.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:509]: This might me confusing to people who don't easily recognize 'violations, needs review and recommendations' as the three categories of issues.  Let's spell it out even more clearly, for example:

 "To <strong>filter</strong> issues and focus only on violations, items to review or recommendations, ...."

Also, the phrase 'press any issue type toggle button in the scan results' will be hard to follow, especially if the issue types don't look like a toggle button.  It would help to rephrase the text to something like:

"... select the issue type in the issue count region below the 'Scan' button."

(That way the text includes the name of the region, so a screen reader user can find it easily.)

Going forward this would be a very good place to add an image.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 2]
```diff
@@ -553,8 +556,7 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
                       >
                         <li>
                           <p style={{ marginTop: "0rem" }}>
-                            Highlight the selected element, or the element
-                            containing the selected issue, in the browser's
+                            Highlight the selected element (purple highlight), in the DOM under the browser's
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:559]: This sentence is the first of two things that happen when you select an element or issue.  It is referring to the highlighting of the element in the DOM view, which is not a purple highlight.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 3]
```diff
@@ -568,14 +570,14 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
                         </li>
                         <li>
                           <p style={{ marginTop: "0rem" }}>
-                            Open and highlight all issues in the element, if any
+                           Any additional issues for the same element, if any, also open and get highlighted
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:573]: This sentence is the second of two things that happen then you select an element or issue, so it starts with the action word 'Open' .  The overall structure of this part of the guide is 
'Select ... to: 
- Highlight something 
- Open something'  

It doesn't work grammatically to start the second bullet with 'Any additional issues ...'
Also, the text has to apply both for the case where the user clicked on a single issue AND also when they selected an element that has multiple issues.  

So overall I feel the original text is better.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 4]
```diff
@@ -568,14 +570,14 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
                             (purple highlight)
                           </p>
                         </li>
                         <li>
                           <p style={{ marginTop: "0rem" }}>
-                            Open and highlight all issues in the element's
-                            child, if any (light purple highlight).
+                            All issues in the element's
+                            child, if any, open and get highlighted (light purple highlight) as well.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:580]: Same comment as above.

**Comment by cpandhi** [comment]
@sharitrewin yes I see your point. Have made the suggested edits. Yes, at some point we will need to update most screenshots and add a few more.

**Comment by cpandhi** [comment]
@sharitrewin, all requested changes have been addressed, please review and approve.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 5]
```diff
@@ -498,16 +498,16 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
                     shown in the ‘Element roles’ view (see the previous
                     definition), while ‘Checklist’ and ‘Rules’ tabs are also
                     available. All views show the same set of issues.
                   </p>
                 </li>
                 <li>
                   <p style={{ marginTop: "0rem" }}>
-                    To <strong>filter</strong> issues by violations, needs review or by recommendations, press any issue type toggle button in the scan results.
-                    Press the same issue type button again to see all issue types.
+                    To <strong>filter</strong> issues and focus only on violations, needs review or by recommendations, select the issue type in the issue count region below the 'Scan' button.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:509]:
```suggestion
                    To <strong>filter</strong> issues and focus only on violations, items that need review or recommendations, select the issue type in the issue count region below the 'Scan' button.
```
(For readability and consistency with line 500/501.)

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 6]
```diff
@@ -570,14 +570,13 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
-                            (purple highlight)
+                           Open and highlight all issues in the element, if any (purple highlight)
                           </p>
                         </li>
                         <li>
                           <p style={{ marginTop: "0rem" }}>
-                            All issues in the element's
-                            child, if any, open and get highlighted (light purple highlight) as well.
+                            Open and highlight all issues in the element's
+                            child, if any (light purple highlight).
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx]:
```suggestion
                            children, if any (light purple highlight).
```

**Comment by cpandhi** [comment]
All changes in, ready for merge.
